.github/workflows/smoke-test-ipv6.yaml

```diff
@@ -0,0 +1,91 @@
+name: Smoke Test with IPv6
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+env:
+  KIND_VERSION: v0.8.1
+  KIND_CONFIG: .github/kind-config-ipv6.yaml
+  CONFORMANCE_TEMPLATE: examples/kubernetes/connectivity-check/connectivity-check-ipv6.yaml
+  TIMEOUT: 2m
+  LOG_TIME: 30m
+
+jobs:
+  conformance-test-ipv6:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Enable IPv6 in docker
+        run: |
+          sudo sh -c "echo '{ \"cgroup-parent\": \"/actions_job\", \"ipv6\": true, \"fixed-cidr-v6\": \"2001:db8:1::/64\" }' > /etc/docker/daemon.json"
+          sudo cat /etc/docker/daemon.json
+          sudo ip -6 route add 2001:db8:1::/64 dev docker0
+          sudo sysctl net.ipv6.conf.default.forwarding=1
+          sudo sysctl net.ipv6.conf.all.forwarding=1
+          sudo systemctl restart docker
+
+      - name: Build docker images
+        run: |
+          make docker-image-no-clean
+          make docker-operator-generic-image
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0-rc.1
+        with:
+          version: ${{ env.KIND_VERSION }}
+          config: ${{ env.KIND_CONFIG }}
+
+      - name: Load local images into kind cluster
+        run: |
+          kind load docker-image --name chart-testing cilium/cilium:latest
+          kind load docker-image --name chart-testing cilium/operator-generic:latest
+
+      - name: Install cilium chart
+        run: |
+          helm upgrade -i cilium ./install/kubernetes/cilium \
+            --namespace kube-system \
+            --set global.nodeinit.enabled=true \
+            --set global.kubeProxyReplacement=partial \
```

Comment:
You can use `--set global.kubeProxyReplacement=strict`. This will allow us to to remove the `nodePort`, `externalIPs`, `hostServices` `hostPort` settings.

Reply:
oh nice, I didn't know about this _strict_ option. Thanks

Reply:
Mmh, I don't think that `kubeProxyReplacement=strict` plays well with `kind`. I'd stick to `partial` for this one.

Reply:
@Rolinh Is it kind or is it the underlying kernel that kind is running on? I'd +1 on `strict` if it works in CI. Not sure what kernel we get there.

Reply:
@joestringer Not sure what the real issue is but if I enable `strict` on a `kind` cluster on my machine (with kernel `5.7.10`) I run into all sort of issues such as coredns pods not getting into ready-state, cilium pod in crashloop trying to reach the kube api server, etc.